README.md

```diff
@@ -377,6 +377,34 @@ okta:
     rateLimit:
       maxRetries: 4
 ```
+
+When you use OAuth 2.0, the full YAML configuration would look like:
+
+```yaml
+okta:
+  client:
+    connectionTimeout: 30 # seconds
+    orgUrl: "https://{yourOktaDomain}" # i.e. https://dev-123456.oktapreview.com
+    proxy:
+      port: null
+      host: null
+      username: null
+      password: null
+    authenticationScheme: "OAUTH2"
```

Comment:
Why is `authenticationScheme` needed? This wasn't defined in our spec.

Reply:
_authenticationScheme_ was something that was internally defined (to be the default) for SSWS case earlier. Now that we have OAUTH2 case, I thought it would be nice to explicitly define it so the caller can specify it in config for clarity. Thoughts?

Reply:
Will get rid of authenticationScheme.